service/src/main/resources/changelog/changesets/20230125_assemble_refbased_io_updates.yaml

```
@@ -0,0 +1,1426 @@
+                          "type": "primitive",
+                          "primitive_type": "Int"
+                        },
+                        "destination": {
+                          "type": "record_update",
+                          "record_attribute": "assembly_length_unambiguous"
+                        }
+                      },
+                      {
+                        "output_name": "assemble_refbased.assembly_mean_coverage",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "Float"
+                        },
+                        "destination": {
+                          "type": "record_update",
+                          "record_attribute": "assembly_mean_coverage"
+                        }
+                      },
+                      {
+                        "output_name": "assemble_refbased.assembly_method",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "String"
+                        },
+                        "destination": {
+                          "type": "none"
+                        }
+                      },
+                      {
+                        "output_name": "assemble_refbased.dist_to_ref_indels",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "Int"
+                        },
+                        "destination": {
+                          "type": "record_update",
+                          "record_attribute": "dist_to_ref_indels"
+                        }
+                      },
+                      {
+                        "output_name": "assemble_refbased.dist_to_ref_snps",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "Int"
+                        },
+                        "destination": {
+                          "type": "record_update",
+                          "record_attribute": "dist_to_ref_snps"
+                        }
+                      },
+                      {
+                        "output_name": "assemble_refbased.ivar_trim_stats",
+                        "output_type": {
+                          "type": "array",
+                          "array_type": {
+                            "type": "map",
+                            "map_type": {
+                              "key_type": {
+                                "type": "primitive",
+                                "primitive_type": "String"
+                              },
+                              "value_type": {
+                                "type": "primitive",
+                                "primitive_type": "String"
+                              }
+                            }
```

It turned out the `map_type` wrapper around key_type and value_type is unnecessary, so we just need key and values when the type is specified as `map`

```suggestion
                            "type": "map",
                            "key_type": {
                              "type": "primitive",
                              "primitive_type": "String"
                            },
                            "value_type": {
                              "type": "primitive",
                              "primitive_type": "String"
                            }
```